### PR TITLE
chore: additional exports for vs code extension

### PR DESCRIPTION
**Pull request by jacobator**
## What/Why/How?
[VS Code extension](https://github.com/Redocly/openapi-vs-code) requires exports of some additional functions

## Check yourself

- [x] Code is linted
- [ ] Tested with redoc/reference-docs/workflows
- [ ] All new/updated code is covered with tests

## Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### CODE REVIEW

File: packages/core/src/index.ts [hunk 1]
```diff
@@ -1,8 +1,16 @@
-export type { BundleOutputFormat } from './utils';
+export { BundleOutputFormat, readFileFromUrl } from './utils';
```
Comment (Marshevskyy) [inline packages/core/src/index.ts:1]: @RomanHotsiy when you get a chance, could you check this change?

**Review by Marshevskyy — APPROVED** [review]
looks good

File: packages/core/src/index.ts (hunk 1 above)
Comment (RomanHotsiy) [reply packages/core/src/index.ts:1]: lgtm

**Review by RomanHotsiy — APPROVED** [review]
LGTM

**Pull request merged**